### PR TITLE
Closing notes on project feast

### PR DESCRIPTION
**Pull request by D4nte**

### CODE REVIEW

File: blog/2020-11-06-project-feast-closing-notes.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+<!--truncate-->
+
+At this point of time, we felt that BTC&lt;&gt;DAI dex trading may not be what users are currently after and hence decided to drop the project.
+
+While the development efforts which lead to this MVP did not lead to a successful product, time was not wasted.
+We made major learnings that will allow us to move faster, for example, our blockchain monitoring solution was not up to scratch for a mainnet environment.
+You can find more details of our learnings in the [closing notes](https://lists.comit.network/pipermail/comit-dev/2020-October/000017.html).
+
+# wBTC&lt;&gt;BTC
```
Comment (bonomat) [inline blog/2020-11-06-project-feast-closing-notes.md:26]: If you don't want to have these weird characters, you could write this: `BTC/DAI` 
```suggestion
# wBTC/BTC
```

Comment (bonomat) [reply blog/2020-11-06-project-feast-closing-notes.md:26]: Actually this looks quite good as it is.

image

File: blog/2020-11-06-project-feast-closing-notes.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+We recently released [ambrosia](./2020-10-12-ambrosia-alpha-release.md), a GUI allowing BTC to DAI peer-to-peer and non-custodial trading powered by COMIT.
+This release marked our change of focus from providing tools to developer to directly targeting end users. 
+
+We spent the last weeks demonstrating this MVP to several users, we had interesting feedback and conversation with various user profiles, from trader to hodler.
+
+<!--truncate-->
+
+At this point of time, we felt that BTC&lt;&gt;DAI dex trading may not be what users are currently after and hence decided to drop the project.
+
+While the development efforts which lead to this MVP did not lead to a successful product, time was not wasted.
```
Comment (bonomat) [inline blog/2020-11-06-project-feast-closing-notes.md:22]: It sounds like a comma is missing: 
```suggestion
While the development efforts, which lead to this MVP, did not lead to a successful product, time was not wasted.
```

File: blog/2020-11-06-project-feast-closing-notes.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+We made major learnings that will allow us to move faster, for example, our blockchain monitoring solution was not up to scratch for a mainnet environment.
+You can find more details of our learnings in the [closing notes](https://lists.comit.network/pipermail/comit-dev/2020-October/000017.html).
+
+# wBTC&lt;&gt;BTC
+
+Several users have expressed interest in a wrapped Bitcoin to Bitcoin atomic swap trading platform.
+The drive behind it is two folds: avoid the option problem and create an open market for ERC20 wrapped BTC.
```
Comment (bonomat) [inline blog/2020-11-06-project-feast-closing-notes.md:29]:
```suggestion
The drive behind it is twofold: avoid the option problem and create an open market for ERC20 wrapped BTC.
```

**Review by bonomat — APPROVED** [review]
LGTM.

File: blog/2020-11-06-project-feast-closing-notes.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+Depositing and withdrawing wrapped Bitcoin can have some restrictions depending on the provider: fees, KYC, high confirmation requirements.
+
+Because of the demand, we forked Ambrosia to provide a wBTC&lt;&gt;BTC version.
+We chose wBTC because it is simply the most popular wrapped Bitcoin token.
```
Comment (thomaseizinger) [inline blog/2020-11-06-project-feast-closing-notes.md:33]: Is this claim actually true?

File: blog/2020-11-06-project-feast-closing-notes.md (hunk 3 above)
Comment (thomaseizinger) [reply blog/2020-11-06-project-feast-closing-notes.md:29]: I am always a bit hesitant with using our internal language within public posts. I doubt that the average user has an idea of what "the option problem" is.

However, I don't think it is worth explaining in this post so feel free to leave it in.

**Review by thomaseizinger — APPROVED** [review]

File: blog/2020-11-06-project-feast-closing-notes.md (hunk 4 above)
Comment (bonomat) [reply blog/2020-11-06-project-feast-closing-notes.md:33]: Yes, if popularity measured by trading volume.

File: blog/2020-11-06-project-feast-closing-notes.md (hunk 3 above)
Comment (D4nte) [reply blog/2020-11-06-project-feast-closing-notes.md:29]: Yes, I thought exactly the same @thomaseizinger but wasn't sure how to handle it :)

**Comment by D4nte** [comment]
Post need to be updated as we are not pursuing on the wBTC<>BTC product. will do this afternoon.

File: blog/2020-11-06-project-feast-closing-notes.md (hunk 2 above)
Comment (D4nte) [reply blog/2020-11-06-project-feast-closing-notes.md:22]: I simplified the sentence.

**Comment by D4nte** [comment]
Updated, will merge by EOD if nobody has an objection

**Pull request merged**